client/src/app/site/projector/components/projector-detail/projector-detail.component.ts

```diff
@@ -353,4 +350,16 @@ export class ProjectorDetailComponent extends BaseViewComponentDirective impleme
             this.messageRepo.create(sendData).then(() => {}, this.raiseError);
         }
     }
+
+    private async loadSubscriptions(): Promise<void> {
```

Comment:
Do not wait until an stable app to subscribe. These subscriptions itself do not keep the spinner up, but some other effects. This is too broad...